### ISSUE TITLE
Need for ontohgis:hasExternalIdentifier

### ISSUE THREAD

**Issue by mereolog** [ontohgis/ontology#18]
Do we still need ontohgis:hasExternalIdentifier?
Originally, it was added to the ontology to keep the links between the ontology and the database, but perhaps it would be a better idea to keep these links in a separate file?

In other words, I would recommend we delete this property (and all its usages).

**Comment by bszady** [comment]
Agree.

pozdrawiam serdecznie
Bogumił

---
dr hab. Bogumił Szady, prof. KUL
orcid.org/0000-0003-0059-5596<http://orcid.org/0000-0003-0059-5596>
---
Ośrodek Badań nad Geografią Historyczną Kościoła w Polsce
Wydział Nauk Humanistycznych
Katolicki Uniwersytet Lubelski Jana Pawła II
---
Institute for the Historical Geography of the Church in Poland
Faculty of Humanities
John Paul II Catholic University of Lublin
---
Al. Racławickie 14, 20-950 Lublin
siedziba (seat): ul. Chopina 29/9
e-mail: ***@***.******@***.***>
e-mail: ***@***.******@***.***>
tel. +48 81 445 31 94
www.kul.pl<http://www.kul.pl/>

________________________________
Od: Pawel Garbacz ***@***.***>
Wysłane: wtorek, 4 lipca 2023 13:33
Do: ontohgis/ontology ***@***.***>
DW: historical.gis ***@***.***>; Assign ***@***.***>
Temat: [ontohgis/ontology] Need for ontohgis:hasExternalIdentifier (Issue #18)


Do we still need ontohgis:hasExternalIdentifier?
Originally, it was added to the ontology to keep the links between the ontology and the database, but perhaps it would be a better idea to keep these links in a separate file?

In other words, I would recommend we delete this property (and all its usages).
